assets/js/modules/analytics-4/datastore/containers.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * `modules/analytics-4` data store: containers.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import Data from 'googlesitekit-data';
+import { MODULES_ANALYTICS_4 } from './constants';
+import { createFetchStore } from '../../../googlesitekit/data/create-fetch-store';
+
+const fetchGetGoogleTagContainerStore = createFetchStore( {
+	baseName: 'getGoogleTagContainer',
+	controlCallback( { measurementID } ) {
+		return API.get(
+			'modules',
+			'analytics-4',
+			'container-lookup',
+			{ destinationID: measurementID },
+			{
+				useCache: false,
+			}
+		);
+	},
+	reducerCallback( state, container, { measurementID } ) {
```

Comment:
We might as well use `createReducer` here too, it would simplify the reducer logic to a one-liner:

```js
	reducerCallback: createReducer( ( state, container, { measurementID } ) => {
		state.containers[ measurementID ] = container;
	} ),
```

Reply:
Updated 👍